data/net/minecraft/client/gui/screens/multiplayer/WarningScreen.mapping

```diff
@@ -0,0 +1,9 @@
+CLASS net/minecraft/client/gui/screens/multiplayer/WarningScreen
+	METHOD <init> (Lnet/minecraft/network/chat/Component;Lnet/minecraft/network/chat/Component;Lnet/minecraft/network/chat/Component;Lnet/minecraft/network/chat/Component;Lnet/minecraft/client/gui/screens/Screen;)V
+		ARG 1 title
+		ARG 2 content
+		ARG 3 checkbox
+		ARG 4 narration
+		ARG 5 lastScreen
```

Comment:
Parameter names which are assigned directly to fields should be named the same as the field.
```suggestion
	METHOD <init> (Lnet/minecraft/network/chat/Component;Lnet/minecraft/network/chat/Component;Lnet/minecraft/network/chat/Component;Lnet/minecraft/network/chat/Component;Lnet/minecraft/client/gui/screens/Screen;)V
		ARG 1 titleComponent
		ARG 2 content
		ARG 3 check
		ARG 4 narration
		ARG 5 previous
```